src/envo/envo-edit.owl

```diff
@@ -44975,15 +44976,30 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605005> "epixylon"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrary.org/obo/ENVO_03605000>)
 
-# Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (stream water)
```

Comment:
I dont understand what happened here - was this class formerly a material ? 

This is a new class right ? Not in current releases ?

Reply:
Looks like @turbomam fixed this